wiki/Coding-Standards.md

```diff
@@ -67,12 +67,15 @@ Another Code Formatting Tool being used is [Prettier](https://prettier.io/). Her
 - "Single Source of Truth" is abbreviated with `Ssot` or `ssot` (don't use `SSoT` or `SSOT`)
 - use `Robot` in all cases instead of `Bot`
 - use `Application` in all cases instead of `App` (in context of supported RPA Applications)
+- use **hyphens** for CSS-classes and CSS-ids consistently
+  For example, don't call the class `buttonBackground` and instead call it `button-background`.
```

Comment:
There is either a dot missing or more indentation is needed.

```suggestion
- use **hyphens** for CSS-classes and CSS-ids consistently
   - For example, don't call the class `buttonBackground` and instead call it `button-background`.
```
Before:
![grafik](https://user-images.githubusercontent.com/38314662/120213695-cc33ce80-c233-11eb-886b-d80232f8e0fb.png)

After: 
![grafik](https://user-images.githubusercontent.com/38314662/120213645-ba522b80-c233-11eb-8273-a6e06b0f750e.png)